src/Microsoft.JSInterop/DotNetDispatcher.cs

```diff
@@ -32,7 +32,7 @@ public static string Invoke(string assemblyName, string methodIdentifier, string
             // the targeted method has [JSInvokable]. It is not itself subject to that restriction,
             // because there would be nobody to police that. This method *is* the police.
 
-            var syncResult = InvokeSynchronously(assemblyName, methodIdentifier, argsJson);
+            var syncResult = InvokeSynchronously(assemblyName, methodIdentifier, /* targetInstance */ null, argsJson);
```

Comment:
if you like this you can just named parameters 😆

Reply:
Yeah, I just always feel weird about it when it's not the final param, because then you have to name the rest even when their names are obvious. But still maybe that's better.